.github/workflows/verify_samples.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Samples
+
+on: [push, pull_request]
+
+jobs:
+  linux:
+    name: Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update -y && sudo apt-get install -y flex bison clang lld
+      - name: Build
+        run: |
+          export PATH=$PATH:/usr/lib/llvm-6.0/bin
+          sh ./.ci_build_samples.sh
+  macOS:
+    name: macOS
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+      - name: Install Dependencies
+        run: |
+          brew update && brew install llvm
+        env:
+          HOMEBREW_NO_INSTALL_CLEANUP: 1
+      - name: Build
+        run: sh ./.ci_build_samples.sh
+  windows:
+    name: Windows
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+      - name: Install MSYS2 & Dependencies
+        run: |
+          choco install msys2
```

Comment:
There is some issue with chocolatey and the way GitHub handles its output, causing thousands of progress messages getting spammed into the log. It can be fixed by replacing this line with `choco install msys2 --no-progress` - I tested this change myself and can confirm that it's working and has no unwanted side effects.

Reply:
Where can I see a log for either variant?

*(I feel like a link to CI logs should have been added to the first post)*

Reply:
The actual branches I used for testing are gone, but the logs are here:
[Without the fix](https://github.com/thrimbor/nxdk/runs/327551909)
[With the fix](https://github.com/thrimbor/nxdk/runs/327572236)

It affects the `Install MSYS2 & Dependencies` part.

Reply:
Yes. Fix should be added. I don't think it hides anything we have to see.

Reply:
Good find! It was rather annoying having my browser window crash when watching the the progress.

Reply:
This has been changed. @GXTX please always leave a comment on discussions *after* pushing a fix (and mention what you changed). This makes it easier for maintainers to know which discussions can be resolved.